README.md

```diff
@@ -15,14 +15,20 @@ Quickstart
 
 Important Notes
 ---------------
-Currently WebAuthn only works in [Firefox's Nightly Build](https://download.mozilla.org/?product=firefox-nightly-latest-ssl&os=osx&lang=en-US)
-After installing:
-1. Open the Firefox advanced preferences at the URL (about:config)[about:config]. These are feature flags for FF Nightly.
+Currently WebAuthn only works in [Firefox's Beta Build](https://download.mozilla.org/?product=firefox-beta-latest-ssl&os=osx&lang=en-US) or [Chrome's Dev 
+Channel](https://www.chromium.org/getting-involved/dev-channel) (U2F tokens only in Chrome)  
+After installing:  
+  
+Firefox:
+1. Open the Firefox advanced preferences at the URL [about:config](about:config). These are feature flags for FF.

```

Comment:
As of 60 Beta, `about:config` edits are no longer needed for Firefox. Both Beta and Nightly "just work." :)

Reply:
huh, I could have sworn I needed to change it when I installed FF Beta, but just confirmed in a VM. I'll update the PR